<launch>
    <!--<group ns="/trollnode">-->
        <node name="setExpression" pkg="trollnode" type="setExpression"/>
		<!-- <node name="publishExpression" pkg="trollnode" type="publishExpression"/> -->
		 <node name="estimateInterest" pkg="trollnode" type="estimateInterest"/> 
		 <node name="greetingNode" pkg="trollnode" type="greetingNode"/> 
		 <node name="lookingNode" pkg="trollnode" type="lookingNode"/> 

    <!--</group>-->
</launch>
